src/libs/SearchParser/searchParser.peggy

```diff
@@ -101,7 +100,6 @@ key
   / "keyword" { return "keyword"; }
   / "sortBy" { return "sortBy"; }
   / "sortOrder" { return "sortOrder"; }
-  / "offset" { return "offset"; }
```

Comment:
We also need to upload the new parser file, no? Or did we get automation to work?

Reply:
I believe we need to update it, will do that but first we need to decided what to do with policyIDs